### ISSUE TITLE
Domain Name

### ISSUE THREAD

**Issue by BenSto** [kjoscha/consulting#31]
siehe Issue Titel. wir brauchen noch einen Namen, der Sinn ergibt und frei ist, für die Domain, die wir kaufen möchten.

Entweder direkt Vorschläge hier oder sonst in einem der nächsten Treffen. Grazie Mille.

**Comment by 00Bock** [comment]
also besser im neuen Repro behandeln, da weniger öffentlich.

**Comment by BenSto** [comment]
Jo, durch den Import wurden allerdings die Issues nicht übernommen. Aber das ist ja nun keine große Hürde. Das Repo hier wird ja nun eh gelöscht.

Weiter geht es dann im private :+1:
